src/main/java/br/com/zup/estrelas/financas/repository/SugestaoService.java

```diff
@@ -0,0 +1,11 @@
+package br.com.zup.estrelas.financas.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import br.com.zup.estrelas.financas.entity.Sugestao;
+
+@Repository
+public interface SugestaoService extends CrudRepository <Sugestao, Long> {

```

Comment:
This one must be a service, not a Repository, u can find an example @ ExampleEntity or in the ones ur mates made

Reply:
In ur service u'll need to code a method for each operation of ur controller